### ISSUE TITLE
Build a pre-production database so we can test new functionality

### ISSUE THREAD

**Issue by atrandafir** [thegooddata/webapp#56]
Setup a backup system of prod database
Copy DB from prod to pre, only users if possible or entire db if not.
Required by Collab - SSO on pre.

**Comment by marcosmenendez** [comment]
It is not a frequent backup. It is a one-off backup that we do once in a while so we can test developments against real data properly.

We should hash all Members PII data and use a stronger username and password than current one for admin.

@atrandafir could you do it manually as @agonbar is proposing or should it better be done by him?

**Comment by atrandafir** [comment]
@marcosmenendez yes I can do it manually but as I commented on 24 Sep 2014 if you look above I have problems restoring the database because on pre the restore command is not installed on the system and if I try to connect from prod where it is installed the prod host is not allowed to connect to pre database, so please @agonbar provide me with a solution :) 

I guess installing pg_restore command in pre would be the fastest, but giving access to connect from prod to pre would also solve it but I do not know at what speed will copy data from one host to another.

**Comment by agonbar** [comment]
But you can import an .sql with the MySQL command directly
mysql -uuser -p database < /dir/file.sql
Or is not installed the MySQL client?

**Comment by atrandafir** [comment]
We are using Postgre for the webapp, forgot about that?

**Comment by agonbar** [comment]
Completely, i'll install the pg_restore command, it should be in an hour.

**Comment by agonbar** [comment]
@marcosmenendez  I'll need a little bit more detailed information on the new issue, thank you.

**Comment by marcosmenendez** [comment]
It is an old issue with a new name. @atrandafir is going to take care of it

**Comment by marcosmenendez** [comment]
We have agreed that @atrandafir will set up a cron job to create fake data daily

**Comment by atrandafir** [comment]
Closing this issue. 

We've got a tool that is generating fake data in pre every day. So we will use that instead of copying production database to pre, mainly for two reasons:  
- production database is getting huge and it would be a pain in the ass to replicate it
- we should not be "playing" with real data in pre, so this way we test with fake data instead of real one

If we finally need to make adjustments to Users table for to have Collaborate Single Sign On system working in Pre then we will fix that manually.
